packages/stream-tile-handler/src/tile-document-handler.ts

```diff
@@ -205,17 +213,61 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
    * @param commitData - Commit to be verified
    * @param context - Ceramic context
    * @param controller - DID value
+   * @param streamId - Stream ID for the commit
    * @private
    */
   async _verifySignature(
     commitData: CommitData,
     context: Context,
-    controller: string
+    controller: string,
+    streamId: StreamID
   ): Promise<void> {
+    const cacao = await this._verifyCapability(commitData, context, streamId)
+
     await context.did.verifyJWS(commitData.envelope, {
       atTime: commitData.timestamp,
       issuer: controller,
       disableTimecheck: commitData.disableTimecheck,
+      capability: cacao,
     })
   }
+
+  /**
+   * Verifies capability attached to a signed commit
+   * @param commitData - Commit to be verified
+   * @param context - Ceramic context
+   * @param streamId - Stream ID for the commit
+   * @returns Cacao is capability was found and verified, null otherwise
+   */
+  async _verifyCapability(
```

Comment:
```suggestion
  async _verifyCapabilityAuthz(
```
I would suggest renaming to make it clearer that we are only authorizing resources here.